polkadot/node/network/statement-distribution/src/v2/tests/requests.rs

```diff
@@ -1356,6 +1358,165 @@ fn v2_descriptors_filtered(#[case] allow_v2_descriptors: bool) {
 		overseer
 	});
 }
+
+#[test]
+fn approved_peer_ump_signal() {
```

Comment:
again some short summary on tested invariants would be really nice.

Reply:
Also seems we are only testing the happy path so far.

Reply:
> Also seems we are only testing the happy path so far.

Because subsystems already check several negative cases for UMP signals. More comprehensive tests are present in the primitives folder, I wouldn't want to duplicate them all in every subsystem that does these checks.